app/assets/stylesheets/components/_important-metadata.scss

```diff
@@ -30,4 +29,8 @@
 
 .app-c-important-metadata__definition {
   font-weight: bold;
+
+  a:focus {
```

Comment:
I think we can use the `govuk-link` class instead?

Reply:
Will do in a separate pr as discussed with Andy